github/doc.go

```diff
@@ -45,12 +45,12 @@ use it with the oauth2 library using:
 		ts := oauth2.StaticTokenSource(
 			&oauth2.Token{AccessToken: "... your access token ..."},
 		)
-		tc := oauth2.NewClient(oauth2.NoContext, ts)
+		tc := oauth2.NewClient(context.Background(), ts)
```

Comment:
Same comment here:
What about adding a line before this:
`ctx := context.Background()`
and then changing this line to use `ctx` ?